editor/src/components/canvas/canvas-strategies/keyboard-absolute-resize-strategy.tsx

```diff
@@ -0,0 +1,146 @@
+import { MetadataUtils } from '../../../core/model/element-metadata-utils'
+import { Keyboard, KeyCharacter } from '../../../utils/keyboard'
+import { CanvasStrategy } from './canvas-strategy-types'
+import { Modifiers } from '../../../utils/modifiers'
+import { CanvasVector } from '../../../core/shared/math-utils'
+import { AdjustCssLengthProperty } from '../commands/adjust-css-length-command'
+import { createResizeCommands } from './shared-absolute-resize-strategy-helpers'
+import { withUnderlyingTarget } from '../../editor/store/editor-state'
+import { EdgePosition } from '../canvas-types'
+import { AbsoluteResizeControl } from '../controls/select-mode/absolute-resize-control'
+
+export const keyboardAbsoluteResizeStrategy: CanvasStrategy = {
+  id: 'KEYBOARD_ABSOLUTE_RESIZE',
+  name: 'Keyboard absolute resize',
+  isApplicable: (canvasState, interactionState, metadata) => {
+    if (canvasState.selectedElements.length > 0) {
+      return canvasState.selectedElements.every((element) => {
+        const elementMetadata = MetadataUtils.findElementByElementPath(metadata, element)
+
+        return elementMetadata?.specialSizeMeasurements.position === 'absolute'
+      })
+    } else {
+      return false
+    }
+  },
+  controlsToRender: [
+    { control: AbsoluteResizeControl, key: 'absolute-resize-control', show: 'always-visible' },
+  ],
+  fitness: (canvasState, interactionState, sessionState) => {
+    if (
+      keyboardAbsoluteResizeStrategy.isApplicable(
+        canvasState,
+        interactionState,
+        sessionState.startingMetadata,
+      ) &&
+      interactionState.interactionData.type === 'KEYBOARD'
+    ) {
+      const { interactionData } = interactionState
+
+      const arrowKeyPressed = interactionData.keysPressed.some(Keyboard.keyIsArrow)
+      const cmdAndOptionallyShiftModifier =
```

Comment:
This mentions `Shift` in the name, but the conditions don't check the shift key?

Reply:
The name wanted to express that it is _exactly_ cmd and optionally shift, so no alt, no ctrl. And the shift is not the in the condition, because it is optional :) I'm open to a better name

Reply:
I think it would have been better to name this to indicate that it is explicitly rejecting `ctrl` and `alt` in that case